src/orion/core/io/space_builder.py

```diff
@@ -88,7 +87,7 @@ def _remove_marker(expression, marker='+'):
     return expression.replace(marker, '', 1) if expression.startswith(marker) else expression
 
 
-class DimensionBuilder(object, metaclass=SingletonType):
```

Comment:
I knew that this day would come :confused:

Reply:
Why are you sad? It's a new day :upside_down_face:  https://www.youtube.com/watch?v=OfJRX-8SXOs